content/applications/finance/fiscal_localizations/italy.rst

```diff
@@ -122,314 +193,349 @@ Accounting.
+     business for personal reasons instead, you have to pay those taxes you originally deducted as
+     a business cost for it.
 
-   .. tab:: `TD07`, `TD08`, and `TD09`
+Odoo may detect that your Vendor Bill can be interpreted as a document of a type that needs Tax
+Integration, as detailed in the :ref:`italy/document-types` section.
 
-      Simplified invoices, and credit/debit notes.
+.. important::
+   Be sure that you replace the :guilabel:`0% Sale Taxes` on the vendor bill you received with the ones
+   you're supposed to pay to the :abbr:`AdE (Agenzia delle Entrate)`. A button then appears on the
+   top of the single vendor bill form to send them.
 
-      Simplified invoices and credit notes can be used to certify **domestic transactions** under
-      **400 EUR** (VAT included). Its status is the same as a regular invoice, but with fewer
-      information requirements.
+   When clicking on the send tax button, an XML file of the appropriate :guilabel:`Document Type` will
+   be generated, attached to the bill and sent as for invoices (put the link of the paragraph you're
+   talking about).
 
-      For a **simplified** invoice to be established, it must include:
+   .. image:: italy/italy-edi-tax-integration-button.png
+      :alt: EDI Send Tax Integration button
 
-      - :guilabel:`Customer Invoice` reference: **unique** numbering sequence with **no gaps**;
-      - :guilabel:`Invoice Date`: issue **date** of the invoice;
-      - :guilabel:`Company Info`: the **seller**'s full credentials (VAT/TIN number, name, full
-        address) under :menuselection:`General Settings --> Companies (section)`;
-      - :guilabel:`VAT`: the **buyer**'s VAT/TIN number (on their profile card);
-      - :guilabel:`Total`: the total **amount** (VAT included) of the invoice.
+.. _italy/document-types:
 
-      In the :abbr:`EDI (Electronic Data Interchange)`, Odoo exports invoices as simplified if:
+Document Types
+--------------
 
-      - It is a **domestic** transaction (i.e., the partner is from Italy);
-      - The buyer's data is **insufficient** for a regular invoice;
-      - The **required fields** for a regular invoice (address, ZIP code, city, country) are
-        provided;
-      - The total amount VAT included is **less** than **400 EUR**.
+The :abbr:`SdI (Sistema di Interscambio)` requires businesses to send customer invoices and other
+documents through the :abbr:`EDI (Electronic Data Interchange)`.
+The following :guilabel:`Document Type` codes all technically identify different business use cases.
 
-      .. note::
-         The 400 EUR threshold was defined in `the decree of the 10th of May 2019 in the Gazzetta
-         Ufficiale <https://www.gazzettaufficiale.it/eli/id/2019/05/24/19A03271/sg>`_. We advise you
-         to check the current official value.
+.. note::
+   We do **not** show the :guilabel:`Document Type` to the user, but it is used in the generation of
+   the XML file.
```

Comment:
once again, not sure why this is interesting here, we give the reader info that can be useful but this seems more of a technical part behind how it works

Reply:
See the discussion with Benj on the other PR for the reasons.
Will remove this sentence for now.